src/app/multi-upload/multi-upload.component.ts

```diff
@@ -21,35 +33,96 @@ export class MultiUploadComponent {
     this.addFileFromDragEvent(<DragEvent> event);
   }
 
+  removeFileBy(id: string, event: Event) {
+    event.preventDefault();
+
+    this.resultFiles = this.resultFiles.filter(file => {
+      const isFileToRemove = file.id === id;
+
+      file.uploadStatus = 'canceled';
```

Comment:
🎉 now you can cancel the upload (FYI: also Google Chrome set the status to canceled)
![image](https://cloud.githubusercontent.com/assets/20790833/26146530/c6da6422-3af0-11e7-9fb3-84f6561221a5.png)